go.mod

```diff
@@ -32,11 +36,14 @@ require (
 	k8s.io/component-base v0.0.0
 	k8s.io/klog v0.3.2
 	k8s.io/kube-openapi v0.0.0-20190401085232-94e1e7b7574c // indirect
+	k8s.io/kubelet v1.15.0
 	k8s.io/kubernetes v1.15.0
 	k8s.io/sample-controller v0.0.0-20190731144349-6f8905ae4ee5
 )
 
 replace (
+	github.com/vmware/go-vmware-nsxt => github.com/MartinWeindel/go-vmware-nsxt v0.0.0-20191129144634-29aa700ba036
```

Comment:
what's the reason for this? was the SDK missing something?

Reply:
good catch, this is not used anymore